public/projects/installfest.md

```diff
@@ -20,15 +20,13 @@ Before the class starts, please install the following:
 ## Node.js (JavaScript)
 
   * **Install** Node.js from <https://nodejs.org/>
-  * version 10.10.0 is preferred
+  * version 10.X.X LTS is preferred
```

Comment:
I know what 10.x.x LTS means, but some students might get confused and look for a version that literally has an "X" in it. You're right that 10.10.0 is a bit old and 10.13.0 is the current preferred version.

Reply:
I mean 10.15.3 is the current preferred version.